core/Utils.py

```diff
@@ -94,3 +116,14 @@ def get_parent_folder(layer_path):
     """
     parts = QgsProviderRegistry.instance().decodeUri('ogr', layer_path)
     return os.path.dirname(parts['path'])
+
+
+def has_point_cloud_provider() -> bool:
+    qgis_providers = QgsProviderRegistry.instance().providerList()
+    point_cloud_providers = QgsProviderRegistry.instance().providersForLayerType(Qgis.LayerType.PointCloud)
```

Comment:
Note this will break for QGIS < v3.30, since `Qgis.LayerType.PointCloud` won't be found.
This would work, though: `QgsProviderRegistry.instance().providersForLayerType(QgsMapLayerType.PointCloudLayer)`

Reply:
BTW, I'll take care of this as well.